prisma/schema.prisma

```diff
@@ -24,3 +24,23 @@ model Password {
   user   User   @relation(fields: [userId], references: [id])
   hash   String
 }
+
+// Originally thought it was going to be a many-to-many relationship
+// Thought about it again and decided it was one-to-many
```

Comment:
Yes! Good job! One group can have many events, but an event is only ever associated with one group!